### PR TITLE
feat(Security): Added info about how to check current versions of fluent bit

### PR DESCRIPTION
**Pull request by paperclypse**
Related to INFOSEC-10264

### CODE REVIEW

File: src/content/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding.mdx [hunk 1]
```diff
@@ -54,6 +54,23 @@ To install the Fluent Bit plugin:
...
+
+Before you upgrade your Fluent Bit plugin, run the following NRQL query to find the current versions of the plugin being used in your system:
+
+```sql
+    FROM
+    K8sContainerSample select latest(containerImage) where podName like '%newrelic-logging%' facet clusterName
+```
+
+<Callout variant="caution">
+    Fluent Bit versions 2.0.7 through 3.0.3 are affected by a [security vulnerability (CVE-2024-4323)](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-4323). If you're using one of these versions, upgrade to a different version. For more info on this, see our [security bulletin NR24-01 - Fluent Bit](https://docs.newrelic.com/docs/security/new-relic-security/security-bulletins/security-bulletin-nr24-01/).
```
Comment (Tcohn1) [inline src/content/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding.mdx]: @paperclypse - the query returns the version of the NR FB output plugin not the FB version. the users need to check if output plugin version they are using is affected: 
[newrelic-fluent-bit-output-1.16.0](https://github.com/newrelic/newrelic-fluent-bit-output/releases/tag/v1.16.0) to [newrelic-fluent-bit-output-1.19.2](https://github.com/newrelic/newrelic-fluent-bit-output/releases/tag/v1.19.2)

Comment (paperclypse) [reply src/content/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding.mdx]: @Tcohn1 I addressed this in my latest commit

File: src/content/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding.mdx [hunk 2]
```diff
@@ -54,6 +54,23 @@ To install the Fluent Bit plugin:
   If you'd rather not compile the plugin yourself, you can download pre-compiled versions from our [GitHub repository's releases page](https://github.com/newrelic/newrelic-fluent-bit-output/releases).
 </Callout>
 
+## Upgrade the Fluent Bit plugin [#upgrade-plugin]
+
+Before you upgrade your Fluent Bit plugin, run the following NRQL query to find the current versions of the output plugin being used in your system:
+
+```sql
+    FROM
+    K8sContainerSample select latest(containerImage) where podName like '%newrelic-logging%' facet clusterName
```
Comment (rhetoric101) [inline src/content/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding.mdx:63]: @paperclypse I'd recommend we use the formatting in our style guide for queries:

FROM  K8sContainerSample 
SELECT latest(containerImage) 
WHERE podName like '%newrelic-logging%' 
FACET clusterName

File: src/content/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding.mdx [hunk 3]
```diff
@@ -54,6 +54,23 @@ To install the Fluent Bit plugin:
...
+```
+
+<Callout variant="caution">
+    Fluent Bit output plugin versions 1.16.0 through 1.19.2 are affected by a [security vulnerability (CVE-2024-4323)](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-4323). If you're using one of these versions, upgrade version 2.0.0 or higher. For more info on this, see our [security bulletin NR24-01 - Fluent Bit](https://docs.newrelic.com/docs/security/new-relic-security/security-bulletins/security-bulletin-nr24-01/).
```
Comment (rhetoric101) [inline src/content/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding.mdx:67]:
```suggestion
    Fluent Bit output plugin versions 1.16.0 through 1.19.2 are affected by a [security vulnerability (CVE-2024-4323)](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-4323). If you're using one of these versions, upgrade to version 2.0.0 or higher. For more information on this, see our [security bulletin NR24-01 - Fluent Bit](/docs/security/new-relic-security/security-bulletins/security-bulletin-nr24-01/).
```

File: src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nr24-01.mdx [hunk 4]
```diff
@@ -85,7 +85,8 @@ New Relic has provided the following resources to assist with these updates:
 * [Install the newest helm charts for the Kubernetes Plugin](https://github.com/newrelic/helm-charts/tree/master?tab=readme-ov-file#Installthecharts)
 * [Install the Kubernetes integration](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/)
 * [Fluent Bit plugin for log forwarding](/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding/)
-
+* [Check the Fluent Bit versions being used on your host](src/content/docs/infrastructure/infrastructure-ui-pages/infrastructure-inventory-page-search-your-entire-infrastructure/#search)
```
Comment (rhetoric101) [inline src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nr24-01.mdx:88]:
```suggestion
* [Check the Fluent Bit versions being used on your host](/docs/infrastructure/infrastructure-ui-pages/infrastructure-inventory-page-search-your-entire-infrastructure/#search)
```

Comment (rhetoric101) [reply src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nr24-01.mdx:88]: @paperclypse although I fixed the relative path, the anchor doesn't seem valid. Could you check this?

**Review by rhetoric101 — COMMENTED** [review]
@paperclypse take a look at a few comments.

File: src/content/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding.mdx (hunk 2 above)
Comment (paperclypse) [reply src/content/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding.mdx:63]: Good call

File: src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nr24-01.mdx (hunk 4 above)
Comment (paperclypse) [reply src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nr24-01.mdx]: I checked it. It links to a collapser. It worked from me in production: https://docs.newrelic.com/docs/infrastructure/infrastructure-ui-pages/infrastructure-inventory-page-search-your-entire-infrastructure/#search

File: src/content/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding.mdx (hunk 3 above)
Comment (paperclypse) [reply src/content/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding.mdx]: Resolved in my latest commit

**Review by rhetoric101 — APPROVED** [review]
@paperclypse this looks good to me!

**Pull request merged**